### ISSUE TITLE
ESM Module Import Error

### ISSUE THREAD

**Issue by ShoGinn** [algorandfoundation/algokit-utils-ts#160]
So, I could be more savvy in this, but from my research, does the ESM version need an extension?

```
(node:94369) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
/mysource/node_modules/.pnpm/@algorandfoundation+algokit-utils@4.1.0_algosdk@2.7.0/node_modules/@algorandfoundation/algokit-utils/esm/index.js:1
import { UpdatableConfig } from './types/config';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at internalCompileFunction (node:internal/vm:77:18)
    at wrapSafe (node:internal/modules/cjs/loader:1288:20)
    at Module._compile (node:internal/modules/cjs/loader:1340:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
    at Module.load (node:internal/modules/cjs/loader:1207:32)
    at Module._load (node:internal/modules/cjs/loader:1023:12)
    at cjsLoader (node:internal/modules/esm/translators:345:17)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:294:7)
    at ModuleJob.run (node:internal/modules/esm/module_job:218:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:329:24)

Node.js v21.1.0
 ELIFECYCLE  Command failed with exit code 1.
 ELIFECYCLE  Command failed with exit code 1.
```

 
 I am getting this error because I use direct injection in my code, and my Algorand class is one of those injected.
 
 I don't think it's a `tsyringe` issue, but the package `discordx` has a function `importx` to import all the commands for my discord bot.

A Link to the importx package. 
https://github.com/discordx-ts/discordx/tree/main/packages/importer

 I intend to refactor my code, but this could still be an issue.

**Comment by neilcampbell** [comment]
@ShoGinn This PR we think should fix the issue https://github.com/algorandfoundation/algokit-utils-ts/pull/161

**Comment by ShoGinn** [comment]
Just loaded the pack in my distro (no changes just pointing to the new kit branch pack) and the errors are eliminated! Look forward to the approval.

**Comment by neilcampbell** [comment]
@ShoGinn The fixes has been released in `5.0.0`

**Comment by ShoGinn** [comment]
This error has returned in `5.1.0` @neilcampbell
